apps/back/src/Controller/ExpenseController.php

```diff
@@ -15,10 +16,22 @@
 #[Route('/expense', name: 'expense')]
 class ExpenseController extends AbstractController
 {
-    #[Route('/list', name: '_list')]
-    public function expensesList(): Response
+    #[Route('/list/{groupName}', name: '_list')]
+    public function expenseList(
+        EntityManagerInterface $em,
+        string $groupName,
+    ): Response
     {
-        return $this->render('expense/list.html.twig', []);
+        $groupRepository = $em->getRepository(Group::class);
+        $group = $groupRepository->findOneBy(['name' => $groupName]);
+
+        if($group !== null) {
+            return $this->render('expense/list.html.twig', [
+                'group' => $group,
+            ]);
+        } else {
+            return new Response(Response::HTTP_NOT_FOUND);
+        }
```

Comment:
Tous les utilisateurs peuvent voir tous les groupes même s'ils n'en font pas parti ? Si seul un membre du groupe peut y accéder, il faudrait ajouter une sécurité ici pour vérifier que l'utilisateur connecté est bien membre et si ce n'est pas le cas, lui interdire l'accés à cette page. Tu peux récupérer l'utilisateur connecté assez facilement avec le bundle Security de Symfony : https://symfony.com/doc/current/security.html#fetching-the-user-object